src/glue/jobs/json_to_parquet.py

```diff
@@ -115,11 +133,17 @@ def get_table(
                 table_name=table_name
             )
     )
+    if table.count() == 0:
+        return table
+    field_names = [field.name for field in table.schema().fields]
     spark_df = table.toDF()
+    if "InsertedDate" in field_names:
```

Comment:
There was an email with Care Evolution. Sorry I forgot to cc you! Here is their words:

> For files that contain "InsertedDate", the "InsertedDate" should be used to identify the most recent values and resolve duplicates.  The "InsertedDate" is included in those files because it is possible for there to be duplicates within the same export file.  For the files without InsertedDate, you should continue to use the date included in the filename to resolve duplicates.